client/src/shared/ui/overlay/Overlay.less

```diff
@@ -84,8 +84,10 @@
   
   fieldset {
     border: none;
-    margin: 0 0 5px 0;
-    padding: 0;
+    margin: 0 -12px 5px 0;
+    padding: 0 12px 0 0;
```

Comment:
Why do we need the negative margin here (adding the padding again)? Is it a CSS hack?

Reply:
I hope my explanation was clear enough to understand

Reply:
Got it, thanks :)